AutomatedTesting/Gem/PythonTests/terrain/utils/FileManagement.py

```diff
@@ -0,0 +1,350 @@
+"""
```

Comment:
What's the purpose of this file?  Is this copied from somewhere else?  If so, can it just be used from that other location?

Reply:
Yes, removed utils folder.